Corona-Warn-App/src/main/res/values-de/covid_certificate_strings.xml

```diff
@@ -102,4 +102,34 @@
     <string name="test_certificate_qrcode_card_sampled_on">"Probenahme am %s Uhr"</string>
     <!-- XTXT: Test certificate card name  -->
     <string name="test_certificate_name">Testzertifikat</string>
+
+    <!-- #####################################################
+      VALIDATION RULES RESULT SCREEN (VALID) STRINGS BELOW
+      ##################################################### -->
+    <!-- XTXT: Validation Rule Result Valid - Header  -->
+    <string name="validation_rules_result_valid_header">Gültiges Zertifikat</string>
+    <!-- XTXT: Validation Rule Result Valid - Result  -->
+    <string name="validation_rules_result_valid_result_title">Ihr Zertifikat ist im gewählten Reiseland gültig.</string>
```

Comment:
Please change "Reiseland" to "Land", as displayed in Figma: https://www.figma.com/file/KuDEcMSREg3VhZRhC3FXTY/COVID19_Master_2.2?node-id=24596%3A21940